player/playloop.c

```diff
@@ -638,6 +639,9 @@ static void handle_pause_on_low_cache(struct MPContext *mpctx)
         use_pause_on_low_cache &= opts->cache_pause_initial &&
                                     (mpctx->video_status == STATUS_READY ||
                                      mpctx->audio_status == STATUS_READY);
+        if (use_pause_on_low_cache) {
+            cache_pause_wait = opts->cache_pause_initial;
+        }
     }
 
     bool is_low = use_pause_on_low_cache && !s.idle &&
```

Comment:
Seems ok, I'd generally prefer if any uses of the changed option field were changed into "val > 0" or so, instead of implicit bool conversion.

Reply:
Done.